docs/tutorials/installation.md

```diff
@@ -1,10 +1,319 @@
+
+# Apple ARM-based GPU (macOS users):
+# Follow Apple's instructions at https://developer.apple.com/metal/jax/
+```
+
+Continue reading this guide to learn about more options.
+
+## Supported platforms
+
+In the table below, check if your setup is supported. And if it says _"yes"_ or _"experimental"_ you can click on the corresponding link to learn how to install JAX in greater detail.
+
+|                  | Linux, x86_64            | Linux, aarch64     | macOS, Intel x86_64        | macOS, ARM-based           | Windows, x86_64 | Windows WSL2, x86_64        |
+|------------------|--------------------------|--------------------|----------------------------|----------------------------|-----------------|-----------------------------|
+| CPU              | [yes](#cpu)              | [yes](#cpu)        | [yes](#cpu)                | [yes](#cpu)                | [yes](#cpu)     | [yes](#cpu)                 |
+| NVIDIA GPU       | [yes](#nvidia-gpu)       | [yes](#nvidia-gpu) | no                         | n/a                        | no              | [experimental](#nvidia-gpu) |
+| Google Cloud TPU | [yes](#google-tpu)       | n/a                | n/a                        | n/a                        | n/a             | n/a                         |
+| AMD GPU          | [experimental](#amd-gpu) | no                 | no                         | n/a                        | no              | no                          |
+| Apple GPU        | n/a                      | no                 | [experimental](#apple-gpu) | [experimental](#apple-gpu) | n/a             |  n/a                        |
+
+Below are just some examples on how to install JAX (do consult with the detailed instructions provided further in this guide):
+
+| Hardware                       | Instructions                                                                                                    |
+|--------------------------------|-----------------------------------------------------------------------------------------------------------------|
+| CPU                            | `pip install -U "jax[cpu]"`                                                                                     |
+| NVIDIA GPU on x86_64 (CUDA 12) | `pip install -U "jax[cuda12_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html`        |
+| NVIDIA GPU on x86_64 (CUDA 11) | `pip install -U "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html` |
+| Google Cloud TPU               | `pip install -U "jax[tpu]" -f https://storage.googleapis.com/jax-releases/libtpu_releases.html`                 |
+| AMD GPU                        | Use [Docker](https://hub.docker.com/r/rocm/jax) or [build from source](https://jax.readthedocs.io/en/latest/developer.html#additional-notes-for-building-a-rocm-jaxlib-for-amd-gpus). |
+| Apple GPU                      | Follow [Apple's instructions](https://developer.apple.com/metal/jax/).    
```

Comment:
This seems duplicative with the first paragraph above. Should we move the table there instead?

Reply:
100%